.prettierrc

```diff
@@ -0,0 +1,10 @@
+{
+  "printWidth": 80,
+  "tabWidth": 2,
+  "semi": true,
+  "singleQuote": false,
```

Comment:
singleQuote as true, please refer to https://github.com/airbnb/javascript#strings--quotes